src/components/menu/styles.ts

```diff
@@ -7,7 +7,7 @@ export const ElMenu = styled.div`
 
 export const ElMenuPopover = styled.div`
   position: absolute;
-  z-index: 98;
+  z-index: 100;
```

Comment:
I just think that since menu will be used on the app bar, it should shown above the drawer